source/develop/release-management/features/storage/incremental-backup.md

```diff
@@ -266,12 +276,20 @@ You need to poll the backup until the phase is `"ready"`.
 Once the backup is ready the response will include `<to_checkpoint_id>` which
 should be used as the `<from_checkpoint_id>` in the next incremental backup.
 
+In case of starting an incremental backup which includes disks that weren't part
+of the given `<from_checkpoint_uuid>`, or new disks that were attached
+to the VM after the given `<from_checkpoint_uuid>` was taken, the system will detect
+those disks and a full backup will be taken for them (`backup_mode="full"`).
+The `backup_mode` for the rest of the the disks will be `incremental` (`backup_mode="incremental"`).
```

Comment:
We are mixing python sdk style (backup_mode="full") and api style <backup_mode>full</backup_mode>.

Lets use only one style, maybe the API since it is more generic and serve all users.